src/resources/dispatch/ReportData.tsx

```diff
@@ -32,13 +32,6 @@ const DispatchDetail = (): React.ReactElement => {
         label='Date'
         source='createdAt'
         labelStyles={{ fontSize: '1rem' }}
```

Comment:
hmm, @salony75way  - this next block of date values were introduced in Taha´s PR, to fix the date:
https://github.com/DeepBlueCLtd/RCO/pull/458/files#diff-2a2b593950593000491a1ba2cbda9cb814687bd0216591ef7bd623f1beccc422R35